### PR TITLE
Update manage-messages.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated from Dismiss to Archive

### CODE REVIEW

**Comment by msbemba** [comment]
Issue # https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2078

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/manage/manage-messages.md [hunk 1]
```diff
@@ -152,53 +152,53 @@ Any message in Message center that is unread will appear in bold. Opening a mess
...
-If you see a message that doesn't pertain to you, or maybe you've already acted on it, you can dismiss the message to remove it from the Message center view. The view that you see in Message center is specific to your user account, so dismissing it from your view doesn't affect other admins.
+If you see a message that doesn't pertain to you, or maybe you've already acted on it, you can archive the message to remove it from the Message center view. The view that you see in Message center is specific to your user account, so archiving it from your view doesn't affect other admins.
 
 ::: moniker range="o365-worldwide"
 
-- On the main page of the message center, select a message, and then select **Dismiss** above the list of messages.
+- On the main page of the message center, select a message, and then select **Archive** above the list of messages.
 
-- Open the message, and then select **Dismiss** at the bottom of the message.
+- Open the message, and then select **archive** at the bottom of the message.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-messages.md]:
```suggestion
- Open the message, and then select **Archive** at the bottom of the message.
```

File: microsoft-365/admin/manage/manage-messages.md [hunk 2]
```diff
@@ -152,53 +152,53 @@ Any message in Message center that is unread will appear in bold. Opening a mess
...
 
 ::: moniker range="o365-germany"
 
-There are two ways to dismiss a message.
+There are two ways to archive a message.
   
 - Select the X that appears to the right of the Message ID. The message is removed from your list. Or,
 
-- Open the message, and then select **Dismiss** at the top of the message.
+- Open the message, and then select **archive** at the top of the message.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-messages.md]:
```suggestion
- Open the message, and then select **Archive** at the top of the message.
```

File: microsoft-365/admin/manage/manage-messages.md [hunk 3]
```diff
@@ -152,53 +152,53 @@ Any message in Message center that is unread will appear in bold. Opening a mess
...
 
 Need to get a dismissed message back? No problem.
   
-1. Select the **Dismissed messages** filter from the **Views** drop-down menu. A list of dismissed messages appears.
+1. Select the **archive messages** filter from the **Views** drop-down menu. A list of archived messages appears.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-messages.md]:
```suggestion
1. Select the **Archived messages** filter from the **Views** drop-down menu. A list of archived messages appears.
```

File: microsoft-365/admin/manage/manage-messages.md [hunk 4]
```diff
@@ -152,53 +152,53 @@ Any message in Message center that is unread will appear in bold. Opening a mess
...
+- Open the message, and then select **Archive** at the top of the message.
 
-Need to get a dismissed message back? No problem.
+Need to get a archived message back? No problem.
   
-1. Select the **Dismissed messages** filter from the **Views** drop-down menu. A list of dismissed messages appears.
+1. Select the **Archived messages** filter from the **Views** drop-down menu. A list of archived messages appears.
 
-2. Select **Restore** to the right of the Message ID, and the message is restored to active messages.
+2. Select **Archive** to the right of the Message ID, and the message is restored to active messages.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-messages.md]:
```suggestion
2. Select **Restore** to the right of the Message ID, and the message is restored to active messages.
```

File: microsoft-365/admin/manage/manage-messages.md [hunk 5]
```diff
@@ -152,53 +152,53 @@ Any message in Message center that is unread will appear in bold. Opening a mess
...
-## Dismiss and restore
+## Archive and restore
 
-If you see a message that doesn't pertain to you, or maybe you've already acted on it, you can dismiss the message to remove it from the Message center view. The view that you see in Message center is specific to your user account, so dismissing it from your view doesn't affect other admins.
+If you see a message that doesn't pertain to you, or maybe you've already acted on it, you can archive the message to remove it from the Message center view. The view that you see in Message center is specific to your user account, so archiving it from your view doesn't affect other admins.
 
 ::: moniker range="o365-worldwide"
 
-- On the main page of the message center, select a message, and then select **Dismiss** above the list of messages.
+- On the main page of the message center, select a message, and then select **Archive** above the list of messages.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-messages.md]:
```suggestion
- On the main page of the Message center, select a message, and then select **Archive** above the list of messages.
```

File: microsoft-365/admin/manage/manage-messages.md [hunk 6]
```diff
@@ -152,53 +152,53 @@ Any message in Message center that is unread will appear in bold. Opening a mess
 
 ::: moniker-end
 
-## Dismiss and restore
+## Archive and restore
 
-If you see a message that doesn't pertain to you, or maybe you've already acted on it, you can dismiss the message to remove it from the Message center view. The view that you see in Message center is specific to your user account, so dismissing it from your view doesn't affect other admins.
+If you see a message that doesn't pertain to you, or maybe you've already acted on it, you can archive the message to remove it from the Message center view. The view that you see in Message center is specific to your user account, so archiving it from your view doesn't affect other admins.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-messages.md]:
```suggestion
If you see a message that doesn't pertain to you, or maybe you've already acted on it, you can archive the message to remove it from the Message center view. The view that you see in the Message center is specific to your user account, so archiving it from your view doesn't affect other admins.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. This is based on issue #2078

Thank you.

**Pull request merged**